src/components/icons/Cogs.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { FunctionComponent } from "react";
```

Comment:
I would change the file structure of these /icons/something.tsx files. Seems to me very repetitive, and contains the same div with img in it. I would create a universal icon component which receives a style, and src prop.